sklearn/feature_selection/_rfe.py

```diff
@@ -70,6 +71,21 @@ class RFE(SelectorMixin, MetaEstimatorMixin, BaseEstimator):
     verbose : int, (default=0)
         Controls verbosity of output.
 
+    importance_getter : str or callable, optional (default='auto')
+        If 'auto', uses the feature importance either through a `coef_`
+        attribute or `feature_importances_` attribute of estimator.
+
+        Also accepts a string that specifies an attribute name/path
+        for extracting feature importance (implemented with `attrgetter`).
+        For example, give `regressor_.coef_` in case of
```

Comment:
referring to `attrgetter` is not essential for our user.

Reply:
Actually, it was mentioned so that user can be aware that they can pass a `attrgetter('regressor_.coef_')` as a param value. 

Plz refer to `test_rfe_wrapped_estimator`.

Reply:
But I agree with you that we can ask users to feed the string value alone and we can put that inside the `attrgetter`. whats your take?

Reply:
Basically a user will never have to write `attrgetter('regressor_.coef_')` but go with the string instead. This is just simpler.

I think the important take with `attrgetter` is that we support a generic callable. And I think that you did a pretty good job regarding the description of the `callable` already.

Reply:
Totally make sense. I wil remove that mention.